File: async-scala-client/src/main/scala/io/swagger/client/model/Payment.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Payment (
  installmentsTerms: Option[Integer],  // Installment terms - 0 # cash - 1 # on terms - 2 # other 
bill: Option[Payment_bill],
installment: Option[List[Payment_installment]],
paymentMode: Option[List[Payment_paymentMode]]
)

File: javascript-closure-angular-client/API/Client/Payment.js
goog.provide('API.Client.Payment');

/**
 * @record
 */
API.Client.Payment = function() {}

/**
 * Installment terms - 0 # cash - 1 # on terms - 2 # other 
 * @type {!number}
 * @export
 */
API.Client.Payment.prototype.installmentsTerms;

/**
 * @type {!API.Client.Payment_bill}
 * @export
 */
API.Client.Payment.prototype.bill;

/**
 * @type {!Array<!API.Client.Payment_installment>}
 * @export
 */
API.Client.Payment.prototype.installment;

/**
 * @type {!Array<!API.Client.Payment_paymentMode>}
 * @export
 */
API.Client.Payment.prototype.paymentMode;

